### ISSUE TITLE
Access to returned list of files possible?

### ISSUE THREAD

**Issue by stueja** [jqueryfiletree/jqueryfiletree#55]
How can I get access to the list of files which is returned by the connector script?

I know that I could run a second `scandir`, however this (still) seems unnecessary and is double work and search time, since `jQueryFileTree.php` has already searched the files.

**Comment by daverogers** [comment]
You could just search the filetree via jquery .find(), right? I'm not really sure what your use case is so that'd be my best guess at the moment.

**Comment by stueja** [comment]
Basically, I want to implement a search field which can search for filenames and return them. I could implement that by running a `scandir` again, but since the files are in the browser already, I consider this unnessesary. 
But, on a second thought, I doubt that the browser already knows all files, but only those which are in the expanded directory, right?

**Comment by daverogers** [comment]
Yeah, that's where it gets sticky. The browser will know about what's been returned via AJAX, but not the stuff that hasn't.
